### ISSUE TITLE
Bug: Interactive Charts - Column as label / row as header issue

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#684]
```
What steps will reproduce the problem? Please provide a link to a
demonstration page if at all possible, or attach code.
1. Create a chart (we tried line and bar graphs) with "column as a label" and/or "row
as a header" checked.
2. Publish chart.
3. Embed chart in html page.
4. "column as a label" and/or "row as a header" commands will not be displayed causing
the chart to not display properly.

<script type="text/javascript" src="//ajax.googleapis.com/ajax/static/modules/gviz/1.0/chart.js">
{"dataSourceUrl":"//docs.google.com/spreadsheet/tq?key=0ArugBYwDrMgacFhlN2dZZV81elRlbUdENFR3enRlYWc&transpose=0&headers=1&range=A1%3AJ13&gid=0&pub=1","options":{"reverseCategories":false,"curveType":"function","backgroundColor":"#FFFFFF","pointSize":"2","width":600,"vAxis":{"format":"'$'#,##0"},"lineWidth":2,"logScale":false,"hasLabelsColumn":true,"hAxis":{"maxAlternation":1},"vAxes":[{"min":null,"title":null,"max":null}],"title":"Austin
Average Homes Sales Price - Aria Realty, Inc.","height":371,"interpolateNulls":false,"legend":"right","reverseAxis":false},"state":{},"chartType":"LineChart","chartName":"Austin
Average Homes Sales Price - Aria Realty, Inc."} </script>


What component is this issue related to (PieChart, LineChart, DataTable,
Query, etc)?
javascript & datatable

Are you using the test environment (version 1.1)?
(If you are not sure, answer NO)
no

What operating system and browser are you using?
Tried Snow Leopard with Chrome, Safari, and FIrefox as well as Windows 7 with IE8


*********************************************************
For developers viewing this issue: please click the 'star' icon to be
notified of future changes, and to let us know how many of you are
interested in seeing it resolved.
*********************************************************

```

Original issue reported on code.google.com by `ariakristen` on 2011-09-09 03:18:11

**Comment by orwant** [comment]
```
The published script looks fine to me.
What exactly do you mean by "commands will not be displayed" in step 4 above?
```

Original issue reported on code.google.com by `ebixon@google.com` on 2011-09-12 13:33:42

**Comment by orwant** [comment]
```
The chart should render as the attached image, but when published, reders
incorrectly: http://ariarealtyaustin.com/test/
It is rendering as if I have not checked "column as a label" and "row as a
header".

<script type="text/javascript" src="//
ajax.googleapis.com/ajax/static/modules/gviz/1.0/chart.js">
{"dataSourceUrl":"//
docs.google.com/spreadsheet/tq?key=0ArugBYwDrMgadDdCamJ0ZGhlYkFoeXA0N0ZrbWFCeFE&transpose=0&headers=1&range=A1%3AI13&gid=0&pub=1","options":{"series":{"0":{"color":"#cc0000"},"1":{"color":"#e69138"},"2":{"color":"#f1c232"},"3":{"color":"#6aa84f"},"4":{"color":"#3d85c6"},"5":{"color":"#674ea7"},"6":{"color":"#a64d79"},"7":{"color":"#ea9999"}},"reverseCategories":false,"curveType":"function","pointSize":"0","backgroundColor":"#FFFFFF","width":626,"vAxis":{"format":"#0.##########"},"lineWidth":2,"logScale":false,"hasLabelsColumn":true,"hAxis":{"maxAlternation":1},"vAxes":[{"min":null,"title":"Months
of Inventory","max":null,"minValue":3,"maxValue":8}],"title":"Months of
Inventory for Austin Homes - Aria Realty,
Inc.","height":366,"interpolateNulls":false,"legend":"right","reverseAxis":false},"state":{},"chartType":"LineChart","chartName":"Months
of Inventory"} </script>

This is happening for all of my charts with the "column as a label" and/or
"row as a header" boxes checked.






On 12 September 2011 08:34,
<google-visualization-api-issues@googlecode.com>wrote:
```

Original issue reported on code.google.com by `ariakristen` on 2011-09-12 13:51:40

**Comment by orwant** [comment]
```
The problem is with the way you encoded the published chart into your page.
All of the '&' were replaced by '&' which caused the wrong data to be retrieved.

If you replace:

"dataSourceUrl":"//docs.google.com/spreadsheet/tq?key=0ArugBYwDrMgadDdCamJ0ZGhlYkFoeXA0N0ZrbWFCeFE&transpose=0&headers=1&range=A1%3AI13&gid=0&pub=1"

with:

"dataSourceUrl":"//docs.google.com/spreadsheet/tq?key=0ArugBYwDrMgadDdCamJ0ZGhlYkFoeXA0N0ZrbWFCeFE&transpose=0&headers=1&range=A1%3AI13&gid=0&pub=1"

The chart should be fine.

```

Original issue reported on code.google.com by `ebixon@google.com` on 2011-09-12 14:15:27

**Comment by orwant** [comment]
```
I see.  This appears to be a Wordpress issue.  Thank you.

On 12 September 2011 09:16,
<google-visualization-api-issues@googlecode.com>wrote:
```

Original issue reported on code.google.com by `ariakristen` on 2011-09-12 14:33:14
